tests/src/JIT/Regression/JitBlue/GitHub_22330/GitHub_22330.cs

```diff
@@ -0,0 +1,49 @@
+using System;
```

Comment:
We probably should add the standard header. I think I added it to the copy in master after the fact.